docs/source/releasing.rst

```diff
@@ -8,8 +8,12 @@ contributions.
 Release checklist
 -----------------
 
-Not all aspects of LiberTEM are covered with automated unit tests. For that
-reason we should perform some manual tests before and after a release.
+* Not all aspects of LiberTEM are covered with automated unit tests. For that
+  reason we should perform some manual tests before and after a release.
+* Handling deprecation, search the code base for :code:`DeprecationWarning`
+  that are supposed to be removed in that release.
+* Add the current LiberTEM version to
+  `CVL <https://github.com/Characterisation-Virtual-Laboratory/CharacterisationVL-Software>`_
```

Comment:
Replace Handling with Handle here. Other than that, the text itself is :ok_hand: 

The steps themselves should be put into the right order, under the "Before" headline below. Like: handling deprecations needs to happen before any QA, and publishing to CVL can only happen once the code is in its final form, that is, if the rest of the release process is done.

Otherwise it looks good! :+1: